index.js

```diff
@@ -25,12 +25,46 @@ module.exports.onRendererWindow = async (window) => {
   if (await isElevated()) {
```

Comment:
This should be the last thing in the method (other than things that depend on it), since this actually stops the current synchronous execution of the method on the `await` and resumes the remainder at some point in the future.

Reply:
Yes, I know. But there is a weird problem with the `window.config.getConfig()` call. It is not working if I call it before the `await isElevated()` call.
I don't want to dig deeper into this because it can be fixed easily when we switch to `decorateHyper`. Then the config is provided as `props` to the component.

Reply:
Very odd. But you're right. Once this plugin is a React HoC decorator, then lots of issues will be solved =]